archey/entries/cpu.py

```diff
@@ -19,6 +19,8 @@ class CPU(Entry):
     Each `dict` **SHOULD** contain only one entry (CPU model name as key and cores count as value).
     """
 
+    _ICON = "\uf4bc"  # oct_cpu 
```

Comment:
```suggestion
    _ICON = "\uf4bc"  # oct_cpu
```

These won't show in most of places (including GitHub, at least on my machine...). I vote for removing them and simply let their "pretty names".